libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/incident/IncidentInfo.kt

```diff
@@ -132,7 +136,8 @@ class IncidentInfo internal constructor(
             "lanesBlocked=$lanesBlocked, " +
             "longDescription=$longDescription, " +
             "lanesClearDesc=$lanesClearDesc, " +
-            "numLanesBlocked=$numLanesBlocked" +
+            "numLanesBlocked=$numLanesBlocked," +
+            "affectedRoadNames=$affectedRoadNames" +
```

Comment:
@RingerJK , do we have a test infrastructure for a DTO which doesn't have a builder?